src/systems/fem_context.C

```diff
@@ -1532,6 +1554,7 @@ void FEMContext::pre_fe_reinit(const System &sys, const Elem *e)
   if (sys.use_fixed_solution)
     this->get_elem_fixed_solution().resize(n_dofs);
   this->get_elem_solution_rate().resize(n_dofs);
+  this->get_elem_solution_accel().resize(n_dofs);
```

Comment:
Are `elem_solution_rate` and (now) `elem_solution_accel` used outside of FEMSystem code?

If not, what about using `dynamic_cast`? If it returns NULL, then it's not FEMSystem and we can safely assume solution_rate etc. aren't being used. If we cast to FEMSystem, then we can grab the time solver and check.